src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Acl/WinGetAssemblyLoadContext.cs

```diff
@@ -41,6 +51,16 @@ private WinGetAssemblyLoadContext()
         /// <returns>The assembly, null if not in our assembly location.</returns>
         internal static Assembly ResolvingHandler(AssemblyLoadContext context, AssemblyName assemblyName)
         {
+            string name = $"{assemblyName.Name}.dll";
+            if (DefaultContextAssemblies.Any(a => a == name))

```

Comment:
This comparison is case sensitive, and this load request could be coming from anywhere.